playbook/app/pb_kits/playbook/pb_message/_message.jsx

```diff
@@ -93,10 +95,12 @@ const Message = (props: MessageProps) => {
             />
           </If>
         </Flex>
-        <Body
-            className="pb_message_body"
-            text={message}
-        />
+        { children || (
```

Comment:
@haydenrou I could imagine use cases where there might be body text **and** children instead of **or**.

Might be worth having the child render out below the Body kit and have both wrapped in conditionals that check for the presence of a prop.

i.e. 
```
<If condition={message}>
  <Body className="pb_message_body" text={message} />
</If>
<If condition={children}>
  {children}
</If>
```

Probably a cleaner way to write the child conditional logic but that'd be the idea.

Reply:
Done, thanks

Reply:
Yes good feedback @bh247484 -- we did the same thing for the Dialog Kit.